producers/src/unit-test/java/uk/gov/moj/sdt/producers/sdtws/SdtEndpointPortTypeTest.java

```diff
@@ -231,6 +293,10 @@ private BulkResponseType createBulkResponse() {
 
         final StatusType statusType = new StatusType();
         statusType.setCode(StatusCodeType.OK);
+        final ErrorType errorType = new ErrorType();
+        errorType.setDescription("MOCK_ERROR");
+        errorType.setCode("MOCK_CODE");
+        statusType.setError(errorType);
```

Comment:
Has this been added to ensure coverage?  If so it should be checked in a test - at the moment it doesn't appear to be checked in any of them.  It also may not be a realistic set of data - can you have a request with a status of OK but also with an error?

The same comment applies to createBulkFeedbackResponse() and createSubmitQueryResponse().